network-api/networkapi/wagtailpages/pagemodels/research_hub/detail_page.py

```diff
@@ -47,6 +47,9 @@ class ResearchDetailLink(wagtail_models.TranslatableMixin, wagtail_models.Ordera
         docs_handlers.DocumentChooserPanel('document'),
     ]
 
+    class Meta(wagtail_models.TranslatableMixin.Meta, wagtail_models.Orderable.Meta):
+        ordering = ['sort_order']
```

Comment:
```suggestion
        pass
```

Reply:
Thanks for the feedback Tibor! In this case, I would vote to stick with defining the sort order, as I see we do this for a lot of other models